build/Codegen.cmake

```diff
@@ -79,19 +79,19 @@ function(generate_bindings_for_kernels)
   execute_process(
     COMMAND
       "${PYTHON_EXECUTABLE}" -c
-      "from distutils.sysconfig import get_python_lib;print(get_python_lib())"
+      "import torchgen;import os; print(os.path.dirname(torchgen.__file__))"
     OUTPUT_VARIABLE site-packages-out
     ERROR_VARIABLE site-packages-out-error
     RESULT_VARIABLE site-packages-result
```

Comment:
Can you change these names to `torchgen-out`, `torchgen-out-error` and `torchgen-result`